### PR TITLE
Update activedirectory.md

### PR DESCRIPTION
**Pull request by sowjanya365**
Issue #1033 - RSAT reference link has been added to the document.

### CODE REVIEW

**Comment by e0i** [comment]
@nenonix @JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: docset/windows/addsadministration/activedirectory.md [hunk 1]
```diff
@@ -24,7 +24,7 @@ ms.assetid: 8AD46969-D3B0-4DE9-81D3-D485B99CB93F
 ## Description
 The Active Directory module for Windows PowerShell is a PowerShell module that consolidates a group of cmdlets. You can use these cmdlets to manage your Active Directory domains, Active Directory Lightweight Directory Services (AD LDS) configuration sets, and Active Directory Database Mounting Tool instances in a single, self-contained package.
 
-If you don't have the Active Directory module installed on your machine, you need to download the correct Remote Server Administration Tools (RSAT) package for your OS.  If you're running Windows 7, you will also need to run the `import-module ActiveDirectory` command from an elevated PowerShell prompt.
+If you don't have the Active Directory module installed on your machine, you need to download the correct Remote Server Administration Tools (RSAT) package for your OS.  If you're running Windows 7, you will also need to run the `import-module ActiveDirectory` command from an elevated PowerShell prompt. For more details on RSAT, see [Remote Server Administration Tools (RSAT) for Windows operating systems](https://support.microsoft.com/help/2693643/remote-server-administration-tools-rsat-for-windows-operating-systems).
```
Comment (nenonix) [inline docset/windows/addsadministration/activedirectory.md]:
```suggestion
If you don't have the Active Directory module installed on your machine, you need to download the correct Remote Server Administration Tools (RSAT) package for your OS.  If you're running Windows 7, you will also need to run the `import-module ActiveDirectory` command from an elevated PowerShell prompt. For more detail, see [RSAT for Windows operating systems](https://support.microsoft.com/help/2693643/remote-server-administration-tools-rsat-for-windows-operating-systems).
```

**Comment by e0i** [comment]
@dariomws306 @get-itips @tseward

Peer review needed for this PR. Thanks.

**Comment by get-itips** [comment]
I am ok with the changes suggested by @nenonix

**Comment by dariomws306** [comment]
Agree

**Comment by e0i** [comment]
@tseward Any opinion on this PR?

**Comment by tseward** [comment]
This needs to be updated for Windows 10 1809+ which includes RSAT OOTB. It can be added via the Manage Optional Features settings pane.

**Comment by sowjanya365** [comment]
Hello Team,

Required fixes are updated in the PR. Kindly let me know if anything pending from my end. 

Thank you.

**Comment by dariomws306** [comment]
Thank you @sowjanya365. @nenonix recommendation is fixed/applied.

I think that Trevor also has a good point here. RSAT on Windows 10 since October 2018 are installed using "Features on Demand", you don't need to download it. Please check for more information:
https://www.microsoft.com/en-us/download/details.aspx?id=45520

Two lines in the same paragraph would be interesting to add.

**Comment by sowjanya365** [comment]
Hi @dariomws306 

Yes, sure. My apologies, I missed the point raised by Trevor. I'm going to add the following sentence to the document. Kindly confirm. 

"Starting with Windows 10 October 2018 Update, RSAT is included as a set of Features on Demand right from Windows 10. Now, instead of downloading an RSAT package you can just go to Manage optional features in Settings and click Add a feature to see the list of available RSAT tools. Select and install the specific RSAT tools you need. To see installation progress, click the Back button to view status on the Manage optional features page."

Thank you.

**Comment by dariomws306** [comment]
Looks fine, thank you!

**Comment by tseward** [comment]
Yep, looks good.

**Comment by e0i** [comment]
@get-itips Agree with the final changes along with @tseward and @dariomws306 ?

**Comment by get-itips** [comment]
@e0i Yes. It's ok.

**Comment by e0i** [comment]
@AndreaBarr The PR that has passed peer review can now be merged. Thanks.

**Pull request merged**